rosdep/python.yaml

```diff
@@ -912,6 +912,21 @@ python-gdown-pip:
   ubuntu:
     pip:
       packages: [gdown]
+python-geographiclib:
+  ubuntu: [python-geographiclib]
+python-geographiclib-pip:
+  debian:
+    pip:
+      packages: [geographiclib]
+  fedora:
```

Comment:
The fedora package is here: https://apps.fedoraproject.org/packages/python-GeographicLib and appears to be available for all recent platforms